frontend/frontend.py

```diff
@@ -197,9 +197,13 @@ def draw_gradio_ui(opt, img2img=lambda x: x, txt2img=lambda x: x, imgproc=lambda
                                                             value=img2img_mask_modes[img2img_defaults['mask_mode']],
                                                             visible=True)
 
-                                    img2img_mask_blur_strength = gr.Slider(minimum=1, maximum=10, step=1,
+                                    img2img_mask_restore = gr.Checkbox(label="Restore image by mask", 
```

Comment:
```suggestion
                                    img2img_mask_restore = gr.Checkbox(label="Restore masked image", 
```
This probably needs a more user friendly name.

Reply:
Good point. I changed it to "Only modify regenerated parts of image".